### DISCUSSION TITLE
error importing opencv-python

### DISCUSSION THREAD

**Discussion by payaldos** [pyodide/pyodide#2424] [General]
I am unable to use newly added packages opencv-python and ffmpeg using  pyodide.loadPackage. I am getting following error

load-package.ts:76 Uncaught (in promise) Error: No known package with name 'opencv-python'
    at addPackageToLoad (load-package.ts:76:11)
    at load-package.ts:111:7
    at Object.<anonymous> (load-package.ts:299:34)
    at Generator.next (<anonymous>)
    at tslib.es6.js:76:71
    at new Promise (<anonymous>)
    at __awaiter (tslib.es6.js:72:12)
    at Object.loadPackage (load-package.ts:288:3)
    at main (html_emotion_recognition.html:18:15)

Am I doing something wrong? Is there an alternative way for this?

**Comment by ryanking13** [comment]
Which version of Pyodide are you using? `opencv-python` will be included in the next release of Pyodide (v0.21.0) and is not included in the latest release (v0.20.0), so you need to use the dev url (https://cdn.jsdelivr.net/pyodide/dev/full/pyodide.js)

**Reply by payaldos** [reply]
Ok Thanks. That works!

**Comment by payaldos** [comment]
Use <script src="https://cdn.jsdelivr.net/pyodide/dev/full/pyodide.js"></script>
to include dev URL in your HTML file.

Further use the following to load pyodide from dev
  let pyodide = await loadPyodide({
          indexURL : "https://cdn.jsdelivr.net/pyodide/dev/full/"
        });

hope this helps

On Sun, Jun 19, 2022 at 12:13 AM Shubham Agrawal ***@***.***>
wrote:

> Hi, I'm very new to python and py-script, So can you please guide me on
> how to change the Pyodide version to the dev URL.?
> I really appreciate any help you can provide.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pyodide/pyodide/discussions/2424#discussioncomment-2977752>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AY6CJORM24DYLRIGGZMBSMLVPYKDJANCNFSM5UUHQE4A>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Payal Doshi
www.prexam.com
www.myclassadmin.com
www.primesoftechsolutions.com
+91-9850553311

**Reply by MeloHenrique** [reply]
But if we are using Pyscript we can't access the imported module inside py-script tag.

**Reply by ryanking13** [reply]
In pyscript, you can use `py-config` tag to specify pyodide URL.

https://github.com/pyscript/pyscript/blob/main/docs/tutorials/getting-started.md#the-py-config-tag

**Reply by shubhamsWEB** [reply]
When I used py-config it is throwing this error:

`Uncaught TypeError: Found non-callable @@iterator
    at PyConfig.connectedCallback (pyconfig.ts:106:34)
    at main.ts:22:34
    at main.ts:35:2`

**Reply by hoodmane** [reply]
That sounds like a downstream bug.

**Reply by MeloHenrique** [reply]
If `py-config` exists that error occurs

**Reply by MeloHenrique** [reply]
The bug has already been shared https://github.com/pyscript/pyscript/issues/528#issue-1276495334
